lib/Settings/Settings.h

```diff
@@ -45,6 +51,8 @@ class Settings {
     // CE and CSN pins from nrf24l01
     cePin(D0),
     csnPin(D8),
+    resetPin(0),
+    radioInterfaceType(PL1167_LT8900),
```

Comment:
Think we should default to NRF, for the sake of backwards compatibility if nothing else.